utils.js

```diff
@@ -1,5 +1,3 @@
-'use strict';
-
 // damn it JS
 export function clone(coll) {
   return JSON.parse(JSON.stringify(coll));
```

Comment:
We also allocate like crazy in Spring. All these will be refactored once things settle down, so don't worry about it for now.